docs/docker.md

```diff
@@ -88,3 +88,53 @@ docker run -d -p 3013:3013 --hostname node0 \
 ```
 
 **Note: make sure `hostname` option is set when reusing the mnesia data directory**
+
+# Localnet
+
+Small local network (not connected to testnet) can be created with `docker-compose`.
+It runs three nodes using the `mean16s-generic` miner (fastest generic miner) and a proxy server to allow CORS.
```

Comment:
Is the usage of mean16 intended for short inter-block time? If yes, this works only for initial ten blocks or so, as after that the difficulty regulation kicks in, with expected 5 minutes inter-block time. See https://github.com/aeternity/epoch/blob/bc08fc66a61a6cd70f8b926107d5e5f5890dc78a/apps/aecore/src/aec_governance.erl#L25-L26 for test-only configuration variable (`expected_mine_rate`) that may be used for shortening inter-block time.

Reply:
The only problem that I see to ship a docker specific sys.config (and change this) is that it will will often go out of sync with the src sys.config. Any idea how to solve this?

Reply:
@dincho 
```
$ ./_build/prod/rel/epoch/bin/epoch start

$ ./_build/prod/rel/epoch/bin/epoch eval 'application:get_env(aecore, expected_mine_rate).'
undefined

$ ./_build/prod/rel/epoch/bin/epoch stop
ok

$ ./_build/prod/rel/epoch/bin/epoch start -aecore expected_mine_rate 100

$ ./_build/prod/rel/epoch/bin/epoch eval 'application:get_env(aecore, expected_mine_rate).'
{ok,100}
```

Reply:
That looks great

Reply:
@lucafavatella I added this option in cea0a63

Reply:
in addition 30b16ce